database/migrate.go

```diff
@@ -0,0 +1,44 @@
+package database
+
+import (
+	"database/sql"
+	"embed"
+	_ "github.com/jackc/pgx/v5/stdlib"
+	"github.com/pkg/errors"
+	"github.com/pressly/goose/v3"
+)
+
+//go:embed migrations/*.sql
+var migrations embed.FS
+
+func Migrate(url string) error {
+	db, err := sql.Open("pgx", url)
```

Comment:
Why is it pgx exactly?
Maybe pass some variable here?

Reply:
I need the standard interface, it just uses a different driver.

What works, we don't touch.